### PR TITLE
Update Cornell voting link in charter.md

### PR DESCRIPTION
**Pull request by annania**
broken url for Cornell voting link. it was adding the ) to the url which made it a Not Found

### CODE REVIEW

**Review by gtback — COMMENTED** [review]
Why not just make it a Markdown link:

`Elections shall be run using the Condorcet-IRV method through the [Cornell online service](http://civs.cs.cornell.edu/).`

😉

File: content/en/charter.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ information, please visit the [TODO Group community & network page](https://todo
 
         11. Elections shall be run using the Condorcet-IRV method
             through the Cornell online service
-            (http://civs.cs.cornell.edu/). The methodology or service
+            (<http://civs.cs.cornell.edu/>). The methodology or service
```
Comment (anajsana) [inline content/en/charter.md:149]:
```suggestion
            [Cornell online service](http://civs.cs.cornell.edu/). The methodology or service
```

**Review by anajsana — CHANGES_REQUESTED** [review]

**Comment by anajsana** [comment]
Good idea @gtback I've requested the changes

**Review by anajsana — APPROVED** [review]

**Comment by anajsana** [comment]
LGTM!

**Pull request merged**